### PR TITLE
petsc: update to 3.22.0

### PR DESCRIPTION
**Pull request by okhlybov**

### CODE REVIEW

File: mingw-w64-petsc/PKGBUILD [hunk 1]
```diff
@@ -145,7 +151,7 @@ _petsc() {
     ;;
     ?s?)
       opts+=" --with-mpi=0 --with-pthread=0 --with-openmp=0"
-      iflags+=" -I\${includedir}/mpiuni"
+      iflags+=" -I\${includedir}/mpiuni"
```
Comment (MehdiChinoune) [inline mingw-w64-petsc/PKGBUILD:154]: What's the purpose of this change?

Comment (okhlybov) [reply mingw-w64-petsc/PKGBUILD:154]: Just a stray space.

Comment (MehdiChinoune) [inline mingw-w64-petsc/PKGBUILD:154]:
```suggestion
      iflags+=" -I\${includedir}/mpiuni"
```
